src/v0/destinations/google_adwords_enhanced_conversions/transform.js

```diff
@@ -58,7 +59,10 @@ const processTrackEvent = async (metadata, message, destination) => {
   let flag = 0;
   const { Config } = destination;
   const { event } = message;
-  const { listOfConversions } = Config;
+  const { customerId, loginCustomerId, listOfConversions } = Config;
+  if (!isString(customerId) || !isString(loginCustomerId)) {
+    throw new InstrumentationError('customerId and loginCustomerId should be a string');
+  }
```

Comment:
This is not correct. We should consider numbers as well as customerId/ loginCustomerId.

Reply:
https://developers.google.com/google-ads/api/rest/reference/rest/v17/customers/uploadConversionAdjustments

so, for This if customerId/loginCustomerId is a number, Then I should convert it to string instead of throwing error. Right?

Reply:
Yes